.github/workflows/Trgr InitDev workflows in Implementation repo.yml

```diff
@@ -57,8 +57,29 @@ jobs:
           $current_branch = '${{env.HEAD_REF}}'
         }
         Write-Host "Current Branch: $current_branch"   
-        echo "current_branch=$current_branch" >> $Env:GITHUB_ENV                   
+        echo "current_branch=$current_branch" >> $Env:GITHUB_ENV
+    - name: Check Ed-Fi-ODS-Implementation Repo PR exists or not
+      shell: pwsh
+      run: |
+        $accessToken = "${{ env.EDFI_ODS_IMP_TOKEN  }}"
+        $apiUrl = "https://api.github.com/repos/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/pulls?state=open&head=Ed-Fi-Alliance-OSS:${{env.current_branch}}"
```

Comment:
Can we use variable substitution to inject the name of the name of the organization and the repository, so that this workflow can run in a fork?